README.md

```diff
@@ -94,7 +108,7 @@ module.exports = function(environment) {
 
 #### `config.featureFlagsService`
 
-The name of the `features` injection can be customized with the `featureFlagsService` config
+The name of the `features` service can be customized with the `featureFlagsService` config
```

Comment:
I believe this feature needs to be deprecated when no longer injecting by default.